wpeverest/user-registration
# UR-341 Enhance - Support links on the checkbox field option

### All Submissions:






### Changes proposed in this Pull Request:


Previously, the Checkbox field option doesn't support any anchor tag link. This PR will fix these issues. 

### How to test the changes in this Pull Request:

1. Navigate to form builder and drag and drop the checkbox field.
2.  In the checkbox option put the link as an option and save the form.
3.  then verify whether it is working as expected or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Support links on the checkbox field option.
